packages/test-utils/src/create-stub.js

```diff
@@ -0,0 +1,7 @@
+export default function createStub (name) {
```

Comment:
I think that it is better to add flow type as below.

```js
export default function createStub (name: string): Component {
```

Reply:
Yes, and add `// @flow` to the top of the file